writeengine/shared/we_config.cpp

```diff
@@ -185,6 +186,17 @@ void Config::checkReload()
     m_BulkRollbackDir += "/rollback";
   }
 
+  const std::string fastDeleteTemp = cf->getConfig("WriteEngine", "FastDelete");
+
+  if (fastDeleteTemp.length() != 0 && boost::iequals(fastDeleteTemp, "true"))
```

Comment:
I have removed the `boost::iequals()` call and now I am only performing a string comparison for either `y` or `Y`. I have also updated the .xml with the default value `n` instead of `false`.